docs/projects/t1-2022/thoth-tech-leadership/weekly-announcements/week5.md

```diff
@@ -0,0 +1,71 @@
+# Week 5 General Announcement
+
+Good Evening Thoth Tech.
+
+Here is this week's update from your friendly neighbourhood leadership team.
+
+First and foremost, congratulations on the excellent feedback on Task 2.1! It
+was a team effort and everyone deserves a pat on the back.
+
+## OnTrack tasks
+
+For D/HD aiming students, you should start working on Task 5.1 retrospective.
+
+If you need some guidance, please watch the D/HD workshop
+[video](https://deakin365.sharepoint.com/:v:/r/sites/ThothTech2/Shared%20Documents/General/Recordings/2-4-22%20D-HD%20Workshop.mp4?csf=1&web=1&e=0Tomse).
+It is beneficial for everyone.
+
+Please look ahead of Task 6.x. There are 4 tasks that due on the same day on Sun
+24/04/2022.
+
+## Company tasks
+
+- Please fill in the learning
+  [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=7Hgj0IgW1UaFQBwotfRw9pxmFQ5mP6pJsCNkD-HdouBUNUVBVlBEMlVTT0RKWkdVNTJOT0o5MTNMTSQlQCN0PWcu)
+  to help us understand the company learning and development needs and plan for
+  resources.
+- Please review the Quality Assurance (QA)
+  [document](https://github.com/thoth-tech/handbook/blob/main/docs/processes/quality-assurance/quality-assurance-overview.md)
+  and start a conversation around the topic with your team. There will be a
+  session on this during the week (Wed 6/04/2022 6:00 PM - 6:45 PM) and it is
+  open to everyone.
+- Please review the Communication
+  [Guide](https://github.com/thoth-tech/handbook/blob/main/docs/communication/communication.md)
+  and see if it can help to streamline your team communication process.
+- Any seniors who want to mentor, please reach out to the Leadership team.
+- Any juniors who would like to work towards leadership next trimester. We are
+  organising an Expression of Interest for attending the leadership conference.
+  Please stay tuned for more information.
+- Please work on the relevant project repositories from **the Thoth Tech
+  organisation**. Please do not pull or push to Doubtfire or SplashKit
+  repositories.
+
+## What you can expect
+
+- Showcase for D/HD students is being organised at the end of the week. We would
+  like to present project Trello boards, plans, testing & data strategies to the
+  Managing Directors. It would be a great opportunity to receive feedback.
+- We are making transition away from MS Word docs to Thoth-tech
```

Comment:
```suggestion
- We are making a transition away from MS Word docs to the Thoth-tech
```

(or 'We are transitioning away')

Reply:
Addressed in [88e2bd0](https://github.com/thoth-tech/handbook/pull/48/commits/88e2bd04ca0081c82c126d66d3d9677d8cdb5a37)